src/Components/PresentationalComponents/CSAwRuleBox/CSAwRuleSummary.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable react/no-danger */
+import React, { Fragment } from 'react';
+import marked from 'marked';
+import PropTypes from 'prop-types';
+import { injectIntl } from 'react-intl';
+import { Link } from 'react-router-dom';
+import sanitizeHtml from 'sanitize-html';
+import { Truncate } from '@redhat-cloud-services/frontend-components/components/Truncate';
+import { Stack, StackItem, TextContent } from '@patternfly/react-core';
+
+import messages from '../../../Messages';
+
+const CSAwRuleSummary = ({ text, truncate, link, intl }) => {
+    const dangerousHtml = (text) => ({ __html: sanitizeHtml(text) });
+
+    return (
+        <TextContent>
+            { truncate
+                ? (
+                    <Truncate
+                        length={230}
+                        expandText={intl.formatMessage(messages.readMore)}
+                        collapseText={intl.formatMessage(messages.readLess)}
+                        text={marked(text)}
+                    />
+                ) : (
+                    <Fragment>
+                        <Stack>
+                            <StackItem>
+                                <span dangerouslySetInnerHTML={dangerousHtml(marked(text))}/>
+                            </StackItem>
+                            <StackItem>
+                                {link && <Link to={`${link}`}>{intl.formatMessage(messages.readMore)}</Link>}
+                            </StackItem>
+
+                        </Stack>
+
+                    </Fragment>
+                )
+            }
+        </TextContent>
+    );
+
+};
+
+CSAwRuleSummary.defaultProps = {
+    truncate: true,
+    link: null
+};
+
+CSAwRuleSummary.propTypes = {
+    intl: PropTypes.any,
```

Comment:
Wouldn't this be better or is the difference negligible?
```
intl: PropTypes.oneOfType([
    PropTypes.string,
    PropTypes.object
  ]),
```

Reply:
explicit is better than implicit :) I will update it